### PR TITLE
Attribution Wizard: Documentation

### PR DESCRIPTION
**Pull request by lennartclaas**
Add section about attribution wizard to USER_GUIDE.md.

Some minor changes in USER_GUIDE.md to reduce oversized images.

Fix: #1299

### CODE REVIEW

File: USER_GUIDE.md [hunk 1]
```diff
@@ -193,14 +193,37 @@ of the signals of the selected resource and the attributions and signals of the
...
-attributions are shown with an `P` icon, even if the relative attributions have been deleted. The cards in the
+attributions are shown with a `P` icon, even if the relative attributions have been deleted. The cards in the
 ` ... in folder content` sub-panels also show the number of resources in the folder that are linked to the shown
 attribution.
 
 The `Add to Attribution Sub-Panel` allows to add an existing attribution to the attributions of the selected resource.
 As in the other panels, the details of the attributions can be shown by clicking on the respective list item, while the
 attribution can be added by clicking on the corresponding **+ icon**.
 
+###### Attribution Wizard
```
Comment (vasily-pozdnyakov) [inline USER_GUIDE.md:204]: Should have a bigger font size.

File: USER_GUIDE.md [hunk 2]
```diff
@@ -193,14 +193,37 @@ of the signals of the selected resource and the attributions and signals of the
...
+
+The `Attribution Wizard` can be opened via the context menu of an assigned attribution. It aggregates information from attributions and signals of the current resource and its descendents and provides a structured overview. By clicking through the wizard, package namespace, name, and version can be selected and saved to the `Attribution Details Column` of the attribution from which the wizard has been opened.
+
+The `Attribution Wizard` has the following structure (compare image):
+
+- Path of the selected resource (1).
+- Breadcrumbs for navigation (2). Each breadcrumb corresponds to a step of the wizard.
+- Package URL (3) that reflects the currently selected package attributes.
+- Lists of aggregated package attributes (4), e.g., package namespace, name, and version (depending on wizard step).
```
Comment (vasily-pozdnyakov) [inline USER_GUIDE.md:213]: no comma after "e.g."

File: USER_GUIDE.md [hunk 3]
```diff
@@ -193,14 +193,37 @@ of the signals of the selected resource and the attributions and signals of the
...
+
+<img src="./docs/user_guide_screenshots/attribution_wizard_doc_overview.png" width="550"/>
+
+In the first step, package namespace and name can be selected. The list entries are sorted by count, i.e., the number of occurrences in the attributions and signals of the current resource and its descendents. New list entries can be added manually via the textboxes below the lists (5). New list items are marked with a star-icon and are displayed at the top of the list (6).
```
Comment (vasily-pozdnyakov) [inline USER_GUIDE.md:217]: no comma after "i.e."
Propose: "New list items are marked"-> "Manually added items are marked..."

**Review by vasily-pozdnyakov — CHANGES_REQUESTED** [review]
Would be nice to update the last three screenshots.

File: USER_GUIDE.md (hunk 1 above)
Comment (lennartclaas) [reply USER_GUIDE.md:204]: True, however, in a logical sense, `Attribution Wizard` must be a subsection of `Attribution Selection Column`, hence it needs one hash more. But if no one has arguments against using one hash less, I can do it. I agree that is would look nicer this way.

File: USER_GUIDE.md (hunk 2 above)
Comment (lennartclaas) [reply USER_GUIDE.md:213]: It depends whether we want to use british (without comma after e.g.) oder american style (with comma after e.g.). I would prefer the american style.

File: USER_GUIDE.md (hunk 3 above)
Comment (lennartclaas) [reply USER_GUIDE.md:217]: Regarding i.e.: see above. 

The proposal is good. I will consider it.

**Comment by lennartclaas** [comment]
@vasily-pozdnyakov regarding you [comment](https://github.com/opossum-tool/OpossumUI/pull/1379#pullrequestreview-1281425912):

What do you mean by that? The last three screenshots are about the attribution wizard, attribution view, and report view. Do you mean updating the last two because their appearance hast changed?

**Comment by lennartclaas** [comment]
I created a new ticket for updating other images in `USER_GUIDE.md`.

**Review by vasily-pozdnyakov — APPROVED** [review]

File: USER_GUIDE.md [hunk 4]
```diff
@@ -193,14 +193,37 @@ of the signals of the selected resource and the attributions and signals of the
...
 ` ... in folder content` sub-panels also show the number of resources in the folder that are linked to the shown
 attribution.
 
 The `Add to Attribution Sub-Panel` allows to add an existing attribution to the attributions of the selected resource.
 As in the other panels, the details of the attributions can be shown by clicking on the respective list item, while the
 attribution can be added by clicking on the corresponding **+ icon**.
 
+##### Attribution Wizard
+
+The `Attribution Wizard` can be opened via the context menu of an assigned attribution. It aggregates information from attributions and signals of the current resource and its descendents and provides a structured overview. By clicking through the wizard, package namespace, name, and version can be selected and saved to the `Attribution Details Column` of the attribution from which the wizard has been opened.
```
Comment (benedikt-richter) [inline USER_GUIDE.md:206]: I'd replace `the context menu of an assigned attribution` by `the context menu of a manual attribution`.

File: USER_GUIDE.md [hunk 5]
```diff
@@ -193,14 +193,37 @@ of the signals of the selected resource and the attributions and signals of the
...
+
+The `Attribution Wizard` has the following structure (compare image):
+
+- Path of the selected resource (1).
+- Breadcrumbs for navigation (2). Each breadcrumb corresponds to a step of the wizard.
+- Package URL (3) that reflects the currently selected package attributes.
```
Comment (benedikt-richter) [inline USER_GUIDE.md:212]: `that reflects the currently selected package attributes.` -> `that describes the currently selected package.`

**Review by benedikt-richter — APPROVED** [review]
Nice documentation! Just added two suggestions.

I worry a bit about the images as these will have to be updated repeatedly (including manually adding the respective numbers (that are displayed in red)).

**Pull request merged**